core/src/main/kotlin/com/malinskiy/marathon/execution/TestBatchResults.kt

```diff
@@ -6,5 +6,14 @@ data class TestBatchResults(
     val device: Device,
     val finished: Collection<TestResult>,
     val failed: Collection<TestResult>,
-    val uncompleted: Collection<TestResult>
-)
+    val uncompleted: Collection<TestResult>,
+    val runCompletionReason: RunCompletionReason
```

Comment:
We discussed that TestBatchResult should have info about the reason why it completes.
Maybe you have better naming purposes